### PR TITLE
T/1 Init repo.

### LINKED ISSUE #1: Init repo

### PR DESCRIPTION
**Pull request by pjasiun**
Closes #1.

### CODE REVIEW

File: tests/_utils/cloudservicesmock.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+
+/**
+ * Mocked CloudServices that
+ */
+export default class CloudServicesMock extends Plugin {
```
Comment (pjasiun) [inline tests/_utils/cloudservicesmock.js]: Do we still need this class?

Comment (ma2ciek) [reply tests/_utils/cloudservicesmock.js]: I removed it.

File: src/cloudservices.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		const options = config.get( 'cloudServices' );
+
+		for ( const optionName in options ) {
+			this[ optionName ] = options[ optionName ];
+		}
+
+		/**
+		 * @readonly
+		 * @member {String} tokenUrl
+		 */
```
Comment (pjasiun) [inline src/cloudservices.js]: There should be added couple words about this config. You can write that it's needed for CS services and give the easy image as an example.

File: src/cloudservices.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+			 * @error cloudservices-token-endpoint-not-provided
+			 */
+			throw new CKEditorError(
+				'cloudservices-token-endpoint-not-provided: The authentication `cloudServices.token` config is not provided.'
+			);
+		}
+
+		/**
+		 * @readonly
+		 */
```
Comment (pjasiun) [inline src/cloudservices.js]: As above, docs are missing. It should be mentioned that other plugins use this token object to authorize and that it handles token refreshing.

File: src/cloudservices.js (hunk 2 above)
Comment (pjasiun) [reply src/cloudservices.js]: Note that also proper docs for the config option need to be added.

**Pull request merged**